semantic/src/site/globals/site.overrides

```diff
@@ -2780,16 +2779,47 @@ h6:first-child {
       padding: 0.5em 0.5em;
     }
     .stage-label {
-      font-size: 0.57rem;
+      font-size: 0.7rem;
+    }    
+  }
+  #overview .left-item {
+    min-width: unset;
+  }
+  #timeline {
+    margin-left: 8px;
+    margin-right: 8px;
+  }
+  
+  #documents-tab {
+    //smaller icon in Review documents tab
+    .ui.image {
+      max-height: 60px;
+    }
+    .flex-extra {
+      gap: 5px;
+    }
+    .ui.list {
+      margin-top: 0px;
+    }
+    .ui.list > .item {
+      margin-top: 10px;
+      padding: 10px;
     }
   }
 
   #notes-tab .options-row {
     flex-direction: column;
+    align-items: flex-start;
   }
 
   #assignment-tab {
     .assigning-row > .column {
+      padding: 0.1rem 0.5rem !important;
+    }
+    .stage-label {
+      margin-left: 3px;
+    }
+    .custom-assignment-grid-column {
```

Comment:
![Screenshot 2023-11-07 at 5 11 03 PM](https://github.com/msupply-foundation/conforma-web-app/assets/5456533/13e640f5-c8a1-4826-9151-2b3c01c2ee98)

I think we need a bit more tidying up of this layout. But lets leave that to another issue.